sdk/schemaregistry/Azure.Data.SchemaRegistry/src/Generated/Models/ContentType.cs

```diff
@@ -1,36 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-// <auto-generated/>
-
-#nullable disable
-
-using System;
```

Comment:
This is a bug in the emitter, this class should not be deleted and wasn't previously: https://github.com/Azure/autorest.csharp/issues/4897